pkg/validation/spec/swagger.go

```diff
@@ -223,6 +286,22 @@ func (s *StringOrArray) UnmarshalJSON(data []byte) error {
 	}
 }
 
+func (s *StringOrArray) UnmarshalNextJSON(opts jsonv2.UnmarshalOptions, dec *jsonv2.Decoder) error {
+	switch k := dec.PeekKind(); k {
+	case '[':
+		return opts.UnmarshalNext(dec, (*[]string)(s))
```

Comment:
Question since I'm curious, what does the spec requires when it comes to multiple unmarshaling? I was experimenting the other day and it felt inconsistent to me.